src/main/java/seedu/address/logic/commands/DeleteSessionCommand.java

```diff
@@ -0,0 +1,68 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_SESSION;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.session.Session;
+import seedu.address.model.session.SessionNumber;
+
+/**
+ * Deletes a session identified using it's displayed index from the session list book.
+ */
+public class DeleteSessionCommand extends Command {
+    public static final String COMMAND_WORD = "deletesession";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the session identified by the session number.\n"
+            + "Parameters: "
+            + PREFIX_SESSION + "SESSION_NUMBER (session must already be created) "
```

Comment:
might want to add a \n after paremeters.
will look cleaner on the UI

Reply:
will do